cmd/server/docs/docs.go

```diff
@@ -24555,7 +24555,7 @@ const docTemplate = `{
     },
     "securityDefinitions": {
         "ApiKeyAuth": {
-            "description": "Custom Token Format, Format: md5('1panel' + 1Panel-Token + 1Panel-Timestamp).\n` + "`" + `` + "`" + `` + "`" + `\neg:\ncurl -X GET \"http://localhost:4004/api/v1/resource\" \\\n-H \"1Panel-Token: \u003c1panel_token\u003e\" \\\n-H \"1Panel-Timestamp: \u003ccurrent_unix_timestamp\u003e\"\n` + "`" + `` + "`" + `` + "`" + `\n- ` + "`" + `1Panel-Token` + "`" + ` is the key for the panel API interface.",
+            "description": "Custom Token Format, Format: md5('1panel' + API-Key + UnixTimestamp).\n` + "`" + `` + "`" + `` + "`" + `\neg:\ncurl -X GET \"http://localhost:4004/api/v1/resource\" \\\n-H \"1Panel-Token: \u003c1panel_token\u003e\" \\\n-H \"1Panel-Timestamp: \u003ccurrent_unix_timestamp\u003e\"\n` + "`" + `` + "`" + `` + "`" + `\n- ` + "`" + `1Panel-Token` + "`" + ` is the key for the panel API Key.",
             "type": "apiKey",
             "name": "1Panel-Token",
             "in": "Header"
```

Comment:
The description of the custom token format has been updated to include an additional term "API-Key," possibly indicating that 1Panel now uses an API key for identification instead of just a standard token. This should be verified against other parts of the documentation and codebase to ensure consistency.

Additionally, there's a potential typo in the URL pattern where it mentions `"http://localhost:4004/api/v1/resource"` instead of using curly braces `{}` which might need adjustment if this path requires dynamic parameter replacement or templating logic elsewhere in the system.

Lastly, while minor formatting changes like indentation can make code readable, they don't impact functionality directly in JSON data (assuming it was originally written as valid JSON).